src/cephadm/cephadm

```diff
@@ -2749,6 +2743,18 @@ def get_ceph_volume_container(ctx: CephadmContext,
     )
 
 
+def set_pids_limit_unlimited(ctx: CephadmContext, container_args: List[str]) -> List[str]:
+    # set containers pid-limits to unlimited rather than defaults (Docker 4096 / Podman 2048)
+    # Useful for daemons like iscsi where the default pid-limit limits the number of luns
+    # per iscsi target or rgw where increasing the rgw thread pool size to a value near
+    # the default pid-limit may cause the container may crash.
+    if isinstance(ctx.container_engine, Podman):
+        container_args.append('--pids-limit=-1')
+    else:
+        container_args.append('--pids-limit=0')
+    return container_args
```

Comment:
I'm not sure I follow the need to return `container_args` here.  Why not
```
# remove extra container args for tcmu container.
# extra args could cause issue with forking service type
tcmu_container.container_args = []
set_pids_limit_unlimited(self.ctx, tcmu_container.container_args)
```
in `get_tcmu_runner_container` and
```
set_pids_limit_unlimited(ctx, container_args)
```
in `get_container`?

Not trying to nit-pick, just curious -- is it considered to be more "pythonic"?

Reply:
yeah, it's unnecessary. Wasn't thinking much about it. Changed it, thanks.